onnx/version_converter/convert.h

```diff
@@ -555,6 +555,9 @@ class DefaultVersionConverter : public BaseVersionConverter {
     registerAdapter(make_unique<CompatibleAdapter>("Scan", OpSetID(18), OpSetID(19)));
     registerAdapter(make_unique<CompatibleAdapter>("Shape", OpSetID(18), OpSetID(19)));
     registerAdapter(make_unique<CompatibleAdapter>("Size", OpSetID(18), OpSetID(19)));
+
+    /******** 19 -> 20 ********/
+    registerAdapter(make_unique<CompatibleAdapter>("GridSample", OpSetID(19), OpSetID(20)));
```

Comment:
You renamed `bilinear` into `linear`. It should be either removed or the converter should handle this case.

Reply:
Thanks @xadupre. I am new to the adapter and will create an adapter for version upgrades in the near future. See the bullet point 3 in the PR description.

Reply:
They are implemented in C++ here: https://github.com/onnx/onnx/tree/main/onnx/version_converter/adapters. You may look for one, see where it is used and replicate the mechanism.